internal/config/config.go

```diff
@@ -16,17 +17,40 @@ type Config struct {
 	Server struct {
 		Port string
 	}
+	OAuth struct {
+		GoogleClientID     string
+		GoogleClientSecret string
+	}
 }
 
 func LoadConfig() (*Config, error) {
 	cfg := &Config{}
-	cfg.Database.Host = getEnv("DB_HOST", "localhost")
-	cfg.Database.Port = getEnv("DB_PORT", "5432")
+	
+	// Add debug logging
+	host := getEnv("DB_HOST", "localhost")
+	fmt.Printf("Database Host: %s\n", host)
+	
+	// Database config
+	cfg.Database.Host = host
+	cfg.Database.Port = getEnv("DB_PORT", "5433")
 	cfg.Database.User = getEnv("DB_USER", "postgres")
 	cfg.Database.Password = getEnv("DB_PASSWORD", "password")
 	cfg.Database.DBName = getEnv("DB_NAME", "kthais")
 	cfg.Database.SSLMode = getEnv("DB_SSLMODE", "disable")
 	cfg.Server.Port = getEnv("SERVER_PORT", "8080")
+	
```

Comment:
Somewhere here you should create two new variables for frontend and backend URL, like cfg.Server.FrontendUrl and cfg.Server.BackendUrl.